components/eventing-controller/pkg/handlers/nats_test.go

```diff
@@ -173,6 +173,8 @@ func TestSubscription(t *testing.T) {
 	}
 }
 
+// TODO should be here and not in "handlers/utils.go"
```

Comment:
Can we just remove [this](https://github.com/kyma-project/kyma/pull/11428/files?file-filters%5B%5D=.go&file-filters%5B%5D=.yaml#diff-930599420e0f1e68cf3283c6d727fa64d563b41c05c2a5e7fd3557d4651af39fR177-R191) instead of this TODO?

Reply:
done